tests/persistence.rs

```diff
@@ -74,41 +73,26 @@ mod persist_tests {
         Ok(())
     }
 
-    // Identical to persist_a, to test concurrency
+    #[test]
+    fn persist_a() -> Result<(), PersistError> {
+        persist()
+    }
 
     #[test]
     fn persist_b() -> Result<(), PersistError> {
-        let n: u64 = 16;
-
-        let mut list = LinkedList::<_, ()>::new();
-
-        for i in 0..n {
-            list.push(i);
-        }
-
-        let persisted = Persistence::persist(&testbackend(), &list)?;
-
-        let restored_generic = persisted.restore()?;
-
-        let mut restored: LinkedList<u64, ()> =
-            LinkedList::from_generic(&restored_generic)?;
-
-        // first empty the original
-
-        for i in 0..n {
-            assert_eq!(list.pop()?, Some(n - i - 1));
-        }
-
-        // then the restored copy
-
-        for i in 0..n {
-            assert_eq!(restored.pop()?, Some(n - i - 1));
-        }
+        persist()
+    }
```

Comment:
You should add a comment here where you explain the reason of those tests, and that we rely on the env to run the tests in threads (e.g. you can explicitly say that having `RUST_TEST_THREADS` set to `1` in the env would make the tests useless, they need multi threads)

Reply:
it's already there, above